pyproject.toml

```diff
@@ -1,5 +1,5 @@
 [build-system]
-requires = ['setuptools>=43', 'wheel', 'setuptools_scm==3.3.3']
+requires = ['setuptools>=43', 'wheel', 'setuptools_scm>=5.0.1']
```

Comment:
I don't trust them to not break anything again.... maybe do `~=5.0.1` instead which will make me feel better but not actually prevent them from breaking again?

Reply:
Sure, we could do that, although we can't use `~=` in `install_requires` AFAIK. Pin to `5.0.x` then?